view/graph/ConceptPreview.kt

```diff
@@ -103,43 +79,67 @@ class ConceptPreview constructor(
         ) { Form.Text(Label.GRAPH_CONCEPT_PREVIEW_PLACEHOLDER, align = TextAlign.Center, softWrap = true) }
     }
 
-    // TODO: copied from TypePage.kt on 23/05/2022
     @Composable
-    private fun TitleSection(concept: Concept) {
+    private fun ConceptTypePreview(concept: Concept) {
```

Comment:
We moved the "title section", which contained the concept icon + type label + attr. value type, into the table, rebranding it as the "Concept Type Preview".